.github/workflows/check-mirrors.yaml

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
This workflow can be removed, It is used to validate mirror & send notification if something breaks!
view here: https://github.com/step-security/agent-api/pull/2696/files#diff-af9ad324e06f7ea5cec44a1110ebc060be501470a2aadc2254b0013238e77044

Also remove `check-mirrors.sh` as well

Reply:
done